web/modules/custom/mymodule/src/Controller/FirstController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file 
+ * Generate markup to be displayed. Functionality in this controller is
+ * wired to Drupal in mymodule.routing.yml.
+ */
+
+namespace Drupal\mymodule\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+
+class FirstController extends ControllerBase {
+
+  public function simpleContent() {
+    return [
+      '#type' => 'markup',
+      '#markup' =>('hello world'),
+    ];
+  }
+
+  public function greetings($name) {
+    return[
+      '#type' => 'markup',
+      '#markup' =>t('HELLO @name !',
+        ['@name' => $name]),
+    ];
+  }
+
+  public function showName() {
+    $user = \Drupal::currentUser();
+    $userName = $user->getAccountName();
```

Comment:
Only class variables will use snakeCase.
For local variables always use camel_case.

Reply:
Yeah I do have used camelCase for local variables and since here was no class variable I haven't used any snake_case. :)